brainscore/submission/repository.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+import subprocess
+import sys
+import zipfile
+from importlib import import_module
+
+import git
+
+from brainscore.submission.configuration import SubmissionConfig, BaseConfig
+from pathlib import Path
+
+from brainscore.submission.models import Submission
+
+logger = logging.getLogger(__name__)
+
+
+def prepare_module(submission:Submission, config:BaseConfig):
+    config_path = config.config_path
+    logger.info('Start executing models in repo submission_%s' % submission.id)
+    repo = extract_zip_file(submission.id, config_path, config.work_dir)
+    package = 'models.brain_models' if submission.model_type == 'BrainModel' else 'models.base_models'
+    return install_project(repo, package)
+
+
+def extract_zip_file(id, config_path, work_dir):
+    zip_file = Path('%s/submission_%s.zip' % (config_path, id))
+    with zipfile.ZipFile(zip_file, 'r') as model_repo:
+        model_repo.extractall(path=str(work_dir))
+    #     Use the single directory in the zip file
+    full_path = Path(work_dir).absolute()
+    return Path('%s/%s' % (str(full_path), find_correct_dir(work_dir)))
+
+
+def find_correct_dir(work_dir):
+    list = os.listdir(work_dir)
+    candidates = []
+    for item in list:
+        if not item.startswith('.') and not item.startswith('_'):
+            candidates.append(item)
+    if len(candidates) is 1:
+        return candidates[0]
+    logger.error('The zip file structure is not correct, we try to detect the correct directory')
+    if 'sample-model-submission' in candidates:
+        return 'sample-model-submission'
```

Comment:
is this for testing only? Can you add an explanatory comment?

Reply:
No this is just trying to guess the correct file in case there are multiple. Has nothing to do with tests, I just thought in case sb didn't rename the folder we can detect it.

Reply:
ok can you please add a short comment saying exactly that?